---a/net/minecraft/block/material/MaterialLiquid.java
+++b/net/minecraft/block/material/MaterialLiquid.java
@@ -1,29 +1,25 @@
 package net.minecraft.block.material;
 
-public class MaterialLiquid extends Material
-{
+public class MaterialLiquid extends Material {
+
     private static final String __OBFID = "CL_00000541";
 
-    public MaterialLiquid(MapColor color)
-    {
+    public MaterialLiquid(MapColor color) {
         super(color);
         this.setReplaceable();
         this.setNoPushMobility();
     }
 
-    public boolean isLiquid()
-    {
+    public boolean isLiquid() {
         return true;
     }
 
-    public boolean blocksMovement()
-    {
+    public boolean blocksMovement() {
         return false;
     }
 
-    public boolean isSolid()
-    {
+    public boolean isSolid() {
         return false;
     }
-}
+}
